app/Actions/Diagnostics/Checks/BasicPermissionCheck.php

```diff
@@ -184,4 +220,47 @@ private function checkDirectoryPermissionsRecursively(string $path, array &$erro
 			Handler::reportSafely($e);
 		}
 	}
+
+	/**
+	 * @throws InvalidConfigOption
+	 */
+	public static function getConfiguredDirectoryPerm(): int
+	{
+		return self::getConfiguredPerm('dir', null, self::FALLBACK_DIRECTORY_PERMS);
+	}
+
+	/**
+	 * @throws InvalidConfigOption
+	 */
+	public static function getConfiguredFilePerm(): int
+	{
+		return self::getConfiguredPerm('file', null, self::FALLBACK_FILE_PERMS);
+	}
+
+	/**
+	 * @param string      $type       either 'dir' or 'file'
+	 * @param string|null $visibility a value out of {@link BasicPermissionCheck::VISIBILITY_CATEGORIES} or `null`
+	 *
```

Comment:
It looks like the description of the `fallbackPermission` parameter is missing?

Reply:
On my local development branch the parameter has already been removed. I tried to fix the bugs you encountered and also to fix the broken tests (even though they run smoothly locally). I am 70% there.

After I will have pushed the next commit, the parameter will be removed here, too.

Reply:
Comment became irrelevant with https://github.com/LycheeOrg/Lychee/pull/1351/commits/b83c6e9736dde70baba995f48bcffd1caf98882f.